src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -208,12 +215,14 @@ export class NotebookContribution extends Disposable implements IWorkbenchContri
 				const copiedInput = this.instantiationService.createInstance(NotebookEditorInput, originalInput.resource, originalInput.name, originalInput.viewType);
 				copiedInput.updateGroup(group.id);
 
-				// transfer ownership of editor widget
-				// const widgetRef = NotebookRegistry.getNotebookEditorWidget(originalInput);
-				// if (widgetRef) {
-				// 	NotebookRegistry.releaseNotebookEditorWidget(originalInput);
-				// 	NotebookRegistry.claimNotebookEditorWidget(copiedInput, widgetRef);
-				// }
+				if (!options?.shouldCreateNewWhenOverride) {
```

Comment:
@bpasero here we check if it's splitting or moving by reading shouldCreateNewWhenOverride, if it's just moving the editor, the new editor input will claim ownership of the NotebookEditorWidget (which is a detached DOM node).
 Write